src/commands/rover/WhoisCommand.js

```diff
@@ -177,6 +206,62 @@ class WhoisCommand extends Command {
         if (Accolades[id]) embed.fields.push({ name: 'Accolades', value: `${Accolades[id]}`, inline: true })
 
         editMessage.edit({ embed: embed }).catch(console.error)
+        
+        let edited = false
+        let scriptingHelpers = ""
```

Comment:
Can you make this code more generic? Instead of having variables for DevForum and SH, make a static list at the top of the file for like "additional resources" or something and dynamically read them.

Reply:
I'm honestly stuck on how to do this without making the code iterate through an array of functions and keep editing the message, which could result in errors down the line.

Reply:
collect the pending changes?